test/unit/resources/aws_iam_password_policy_test.rb

```diff
@@ -66,6 +66,13 @@ def test_number_of_passwords_to_remember_returns_configured_value
     assert_equal expectedValue, AwsIamPasswordPolicy.new(@mockConn).number_of_passwords_to_remember
   end
 
+  def test_policy_to_s
+    configure_policy_password_reuse_prevention(value: Object.new)
+    expectedValue = "IAM Password-Policy"
+    test = AwsIamPasswordPolicy.new(@mockConn).to_s[0..expectedValue.length - 1]
```

Comment:
I think this is a relic from when we didn't really know if there would be some kind of an ID returned along with the string. Ie, "IAM Password-Policy <id>". Now that we know that there _won't_ be, we don't need to do any substringing.